training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.5-gpu/context/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu115-py38-torch1110:{{latest-image-tag:monthly\.\d{6}}}
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.0' \
```

Comment:
do you want to have it in requirements.txt file? 
any reason to break pip install into 2 parts?
conda yaml should be removed probably since it's empty
why do we have the same environment under different name in internal repo and here?

Reply:
removed empty conda.yaml
We would remove env from internal repo once this works and checked in.
Broke pip install into two parts to have clarity on training vs model scoring apis.